docs: update FPF slot information for Ulmo (#1336)
openedx/docs.openedx.org: source/community/release_notes/ulmo/ulmo_frontend_plugin_slots.rst

.. _Ulmo Frontend Plugin Slots:

New Frontend Plugin Slots for the Ulmo Release
###############################################

Utilizing *frontend plugin slots*, site operators now have the ability to
customize various portions of the site.

A “frontend plugin slot” refers to an area of a web page - comprising one or
more visual elements - that can be “swapped out” with other visual elements. For
example, one plugin slot allows you to remove the "Help" button in the header.

Overhead and hassle is minimized using the plugin slot system. Site operators
can leverage a plugin slot using a configuration file; the codebase does not
need to be copied (“forked”) nor are extensive changes needed. A snippet of code
is all that is needed to use a plugin slot. A site operator places that code
within a configuration file. Site operators should refer to the
``src/plugin-slots`` directory within each MFE's codebase to view documentation
for that MFE's plugin slot(s).

In these release notes, we'll detail the new and updated plugin slots found in the Ulmo
release. The full list of existing plugin slots can be found in

> :doc:`/site_ops/references/frontend-plugin-slots`, and usage instructions are

Reviewer: I think you probably copied this, but we should use `:ref:` syntax as it's less brittle. Here it would be 

```suggestion
:ref:`Available Frontend Plugin Slots`, and usage instructions are
```

since that's the ref atop the page on https://raw.githubusercontent.com/openedx/docs.openedx.org/refs/heads/main/source/site_ops/references/frontend-plugin-slots.rst

> available at :doc:`/site_ops/how-tos/use-frontend-plugin-slots`.

Reviewer: ```suggestion
available at :ref:`Use A Frontend Plugin Framework Slot`.
```

  This slot is used for modifying "View Courses" button in the course exit screen

* Added `DashboardFootnoteLinkPluginSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/CourseExitPluginSlots/DashboardFootnoteLinkPluginSlot>`_:
  This slot is used for modifying the link to the learner dashboard in the footnote on the course exit page

* Added `SequenceNavigationSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/SequenceNavigationSlot>`_:
  This slot is used to replace/modify/hide the sequence navigation component that controls navigation between units within a course sequence.

* Updated `CourseBreadcrumbsSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/CourseBreadcrumbsSlot>`_:
  Added ``pluginProps``.

* Updated `NotificationTraySlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/NotificationTraySlot>`_:
  Added ``pluginProps``.

* Updated `UnitTitleSlot <https://github.com/openedx/frontend-app-learning/tree/release/ulmo/src/plugin-slots/UnitTitleSlot>`_:

>   Modified ``pluginProps`` behavior, ``isEnabledOutlineSidebar`` is now always `true`. 

Reviewer: ```suggestion
  Modified ``pluginProps`` behavior, ``isEnabledOutlineSidebar`` is now always ``true``. 
```

.. seealso::

   * :doc:`/site_ops/how-tos/use-frontend-plugin-slots`

   * :doc:`../sumac/customizing_learner_dashboard`

   * :doc:`../sumac/customizing_header`

**Maintenance chart**

+--------------+-------------------------------+----------------+--------------------------------+
| Review Date  | Working Group Reviewer        |   Release      |Test situation                  |
+--------------+-------------------------------+----------------+--------------------------------+
|              | Frontend WG                   | Ulmo           |                                |

> +--------------+-------------------------------+----------------+--------------------------------+

Author: This is the main reason this PR is a draft.

Reviewer: Gotcha. I would put today's date (it's accurate as of today) and either your name or "Frontend WG"
